apps/meteor/app/models/server/models/Subscriptions.js

```diff
@@ -12,8 +12,8 @@ export class Subscriptions extends Base {
 	constructor(...args) {
 		super(...args);
 
-		this.tryEnsureIndex({ rid: 1 });
```

Comment:
how are we going to remove this index later on? 👀

Reply:
I have the migration, but migrations are a sin now, so not sure if I should sin :trollface:

Reply:
we got rid of this already 😬